docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -34,11 +34,31 @@ ifeval::["{build}" != "downstream"]
 CONTROLLER1_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100"
 endif::[]
 ifeval::["{build}" == "downstream"]
-CONTROLLER1_SSH="ssh -i *<path to SSH key>* root@*<controller-1 IP>*"
-endif::[]
+CONTROLLER1_SSH="ssh -i *<path to SSH key>* root@*<controller-1 IP>*" <1>
 CONTROLLER2_SSH="ssh -i *<path to SSH key>* root@*<controller-2 IP>*"
 CONTROLLER3_SSH="ssh -i *<path to SSH key>* root@*<controller-3 IP>*"
+endif::[]
 ----
+* Specify IP addresses of all controllers, for example:
++
+[subs=+quotes]
+----
+ifeval::["{build}" != "downstream"]
+CONTROLLER1_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.103"
+CONTROLLER2_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.106"
+CONTROLLER3_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.109"
+endif::[]
+ifeval::["{build}" == "downstream"]
+CONTROLLER1_SSH="ssh -i *<path to SSH key>* root@*<controller-1 IP>*" <2>
+CONTROLLER2_SSH="ssh -i *<path to SSH key>* root@*<controller-2 IP>*"
+CONTROLLER3_SSH="ssh -i *<path to SSH key>* root@*<node3 IP>*"
+# ...
+endif::[]
+----
+ifeval::["{build}" == "downstream"]
+<1> Replace `<path_to_SSH_key>` with the path to your SSH key.
+<2> Replace `<controller-X IP>` with IP addresses of all controllers.
```

Comment:
```suggestion
<2> Replace `<controller-1 IP>` with the IP addresses of all Controllers.
```
The value to replace here should match the line of code.

Reply:
In code there is
```
ifeval::["{build}" == "downstream"]
CONTROLLER1_SSH="ssh -i *<path to SSH key>* root@*<controller-1 IP>*" <2>
CONTROLLER2_SSH="ssh -i *<path to SSH key>* root@*<controller-2 IP>*"
CONTROLLER3_SSH="ssh -i *<path to SSH key>* root@*<node3 IP>*"
# ...
endif::[]
```
isn't that matches to <controller-X IP>?

Reply:
@klgill sorry, I meant *this* renders OK for me

Reply:
Ah, I see. You used <controller-x-IP> as a "catch-all" for each value.
Typically, our style is to list each env var. But you could probably simplify it by using only one bullet point, for example:

* Replace <controller-1 IP>, <controller-2 IP>, and <node3 IP> with the IP addresses of each Controller node.